app/helpers/database/index.ts

```diff
@@ -18,8 +18,8 @@ export function extractChannelDisplayName(raw: Pick<Channel, 'type' | 'display_n
             displayName = raw.display_name.trim() || record?.displayName || '';
             break;
         case General.GM_CHANNEL: {
-            if (raw.fake) {
-                displayName = raw.display_name;
+            if (raw.fake || raw.display_name) {
```

Comment:
what are you doing here? this will cause problems with what we had previously

Reply:
So before this change, the GM displayName was not being updated as this function was returning the previous displayName...not the computed one.

Reply:
and so what happens to what we had? did you considered that? what is the fallout of this change?

don;t you think is best to set the `fake` property to true when you actually know this value indeed changed?